src/lib/FormGroupLayoutLabel.tsx

```diff
@@ -42,6 +42,8 @@ const FormGroupLayoutLabel = <T extends FieldValues>(props: FormGroupLayoutLabel
             stroke="currentColor"
             id={`Tooltip-${fieldId}`}
             className="tooltip--icon"
+            width="24"
```

Comment:
@Harsh517-tech415 Can you please add a prevent default onClick please?

When we are using it with a switch / checkbox, I would avoid to toggle onClick :-)